docs/_templates/2018/na/menu-mobile.html

```diff
@@ -33,6 +33,7 @@
                           <li><a href="{{ pathto('conf/portland/2018/convince-your-manager') }}">Convince Your Manager</a></li>
                           <li><a href="{{ pathto('conf/portland/2018/welcome-wagon') }}">Welcome Wagon</a></li>
                           <li><a href="{{ pathto('conf/portland/2018/team') }}">Meet the Team</a></li>
+                          <li><a href="{{ pathto('conf/portland/2018/speaking-tips') }}">Speaking Tips</a></li>
```

Comment:
Should this really be a public website menu? Feels overly specific to include on the actual conf menu.

Reply:
To my mind, I like it in the conference menu, since it's an attendee resource, and I think About's the best place for it. If you think about 'Convince your manager' as a resource for a specific subset of attendees, then this could fall kinda under that category too. 

(it's not perfect, and I do think it might eventually be best to have it over on www, but I'm with Sam for now – I think the brand difference is real jarring)